app/Http/Controllers/ReportsController.php

```diff
@@ -378,6 +380,9 @@ public function getActivityReportDataTable()
                 } else {
                     $activity_target = '';
                 }
+            } elseif (($activity->action_type=='accepted') || ($activity->action_type=='declined')) {
+                $activity_target = '<a href="' . route('view/user', $activity->item->assigneduser->id) . '">' . e($activity->item->assigneduser->fullName()) . '</a>';
+
```

Comment:
Separate from this... but I'm thinking maybe we want to go towards a Helper::generateHtmLinkFromItem($item) that does something like

```
return "a href=" . route("view/${metaprogramming_classname_somethigorother_from_item}", $item->id) . ">" . $item->displayableName . "</a>"
```
because we seem to do this so often... not sure if this is the best way though.. thoughts?